docs/examples/tooltip/poper-locations.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="play-container">
+    <el-tooltip
+      :popper-options="popperOptions"
+      class="box-item"
+      popper-class="popper-tooltip"
+      :show-arrow="false"
+      effect="dark"
+      placement="top-start"
+      content="popper-options has a problem"
+    >
+      <el-button>Custom Popper Location</el-button>
+    </el-tooltip>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { reactive } from '@vue/reactivity'
```

Comment:
import { reactive } from 'vue'

Reply:
> Hello @Zourunfa, thank you for contributing to element-plus, please see our [guideline](https://github.com/element-plus/element-plus/blob/dev/CONTRIBUTING.md) to see how to make contribution



> @Zourunfa First of all, thank you for your contribution, but what happened to the 2 deleted files?

because ,I want to remove console.log ,but  但是不小心删除文件了

Reply:
> import { reactive } from 'vue'

done